app/src/main/java/uk/gov/android/securestore/crypto/RsaHybridCryptoManager.kt

```diff
@@ -26,39 +31,43 @@ internal class RsaCryptoManager : CryptoManager {
         this.alias = alias
     }
 
-    override fun encryptText(
-        text: String,
-    ): String {
+    override fun encrypt(
+        input: String,
+        callback: (key: ByteArray) -> String?,
```

Comment:
Think we need to rework this setup with the interfaces, not sure if we should be having an 'unused' callback here

Reply:
But then it would be two different interfaces for the same purpose - the only alternative way is to return a key from encrypt instead of using the call back, I think? Unless there's a way you have in mind?